UML-Diagram/ReturnSystem.puml

```diff
@@ -0,0 +1,28 @@
+@startuml ReturnSystem
+
+package "Return System" {
+    interface IReturnStrategy {
+        + ReturnProcess(Book[] books)
+    }
+
+    class ReturnContext {
+        - borrow : Borrow
+        - fine : Fine
+        - returnStrategy : IReturnStrategy
+        + ReturnContext(IReturnStrategy returnStrategy)
+        + setReturnStrategy(IReturnStrategy returnStrategy) : void
+        + performReturn() : void
+    }
+
+    class RegularReturnStrategy implements IReturnStrategy {
+        + ReturnProcess(Book[] books)
```

Comment:
Generate two lists but there are never used. The class Return context its not used on other classes, so ithink its not used  in other parts of the program.